src/napari_imagej/widget_IJ2.py

```diff
@@ -0,0 +1,124 @@
+from enum import Enum
+from typing import List
+
+from magicgui.widgets import request_values
+from napari import Viewer
+from napari.layers import Layer
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QIcon, QPixmap
+from qtpy.QtWidgets import QMessageBox, QPushButton, QVBoxLayout, QWidget
+
+from napari_imagej._module_utils import _get_layers_hack
+from napari_imagej.setup_imagej import ij, jc, running_headless
+
+
+class GUIWidget(QWidget):
+    def __init__(self, viewer: Viewer):
+        super().__init__()
+        self.setLayout(QVBoxLayout())
+
+        self.gui_button: GUIButton = GUIButton()
+        self.layout().addWidget(self.gui_button)
+
+        self.to_ij: ToIJButton = ToIJButton()
+        self.layout().addWidget(self.to_ij)
+
+        self.from_ij: FromIJButton = FromIJButton(viewer)
+        self.layout().addWidget(self.from_ij)
+        if not running_headless():
+            self.gui_button.clicked.connect(lambda: self.to_ij.setHidden(False))
+            self.gui_button.clicked.connect(lambda: self.from_ij.setHidden(False))
```

Comment:
Do we really need to prevent these from appearing until we click the first button? Or can we just disable them until the button is clicked?

Reply:
Now we do the latter!